PhysicsTools/PatAlgos/python/slimming/slimmedOOTPhotons_cff.py

```diff
@@ -0,0 +1,14 @@
+import FWCore.ParameterSet.Config as cms
+from PhysicsTools.PatAlgos.slimming.slimmedPhotons_cfi import *
+
+slimmedOOTPhotons = slimmedPhotons.clone()
+slimmedOOTPhotons.src = cms.InputTag("selectedPatOOTPhotons")
+
+slimmedOOTPhotons.linkToPackedPFCandidates = cms.bool(False)
+slimmedOOTPhotons.recoToPFMap = cms.InputTag("")
+slimmedOOTPhotons.packedPFCandidates = cms.InputTag("")
+slimmedOOTPhotons.saveNonZSClusterShapes = cms.string("(r9()>0.8)") # save additional user floats: (sigmaIetaIeta,sigmaIphiIphi,sigmaIetaIphi,r9,e1x5_over_e5x5)_NoZS 
+#slimmedOOTPhotons.saveNonZSClusterShapes = cms.string("(r9()>0.8 || hcalTowerSumEtConeDR04()<20 || hcalTowerSumEtConeDR04()<0.3*pt())") # save additional user floats: (sigmaIetaIeta,sigmaIphiIphi,sigmaIetaIphi,r9,e1x5_over_e5x5)_NoZS 
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant

If really needed here
it would help to make this shorter by moving ```# save additional user floats: (sigmaIetaIeta,sigmaIphiIphi,sigmaIetaIphi,r9,e1x5_over_e5x5)_NoZS``` above the two (one in, one commented out) lines